superchain/implementations/networks/sepolia.yaml

```diff
@@ -4,22 +4,28 @@ l1_erc721_bridge:
   2.1.0: 0xAE2AF01232a6c4a4d3012C5eC5b1b35059caF10d
 l1_standard_bridge:
   2.1.0: 0x64B5a5Ed26DCb17370Ff4d33a8D503f0fbD06CfF
-l2_output_oracle:
-  1.8.0: 0xF243BEd163251380e78068d317ae10f26042B292
```

Comment:
We still have a list of "global" L2OO implementations defined in `implementations.yaml` [here](https://github.com/ethereum-optimism/superchain-registry/blob/5b11cbf3f63d9b66259d1120126403fa36cd7ca6/superchain/implementations/implementations.yaml#L13-L16) that is [merged into the network config here](https://github.com/ethereum-optimism/superchain-registry/blob/f68cf836cb872913b5a175c999dd4b566fd1b886/superchain/superchain.go#L582).  Given we still have L2OO impls defined, just wanted to check that it makes sense to remove the implementation here?

Reply:
I lean towards removing this - nothing should use it anymore so we shouldn't need to keep a record of the custom location this version wound up on for sepolia.